hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3,18 +3,1146 @@
+            verified by the SW test for correctness. Testbench then reads and verifies
+            a known data. SW validates the reception of TBD interrupts (the SW test / testbench
+            create those scenarios).
+            Verify all instances of I2C in the chip independently.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // USB (pre-verified IP) integration tests:
+    {
+      name: chip_usb_fs_tx_rx
+      desc: '''Verify the transmission of data over the USB at full speed.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_usb_vbus
+      desc: '''Verify that the USB device can detect the presence of VBUS from the USB host.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_usb_suspend
+      desc: '''Verify that the USB device can detect the presence of VBUS from the USB host.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_usb_pullup
+      desc: '''Verify the enablement of USB device pullup.
+
+            TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // PINMUX (pre-verified IP) integration tests:
+    {
+      name: chip_pin_mux
+      desc: '''Verify the MIO muxing at input and output sides.
+
+            SW programs MIO INSEL and OUTSEL CSRs to connect and verify each muxed source. At the
+            moment, GPIOs are the only mux inputs.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sleep_pin_mio_dio_val
+      desc: '''Verify the MIO output values in deep sleep state.
+
+            SW programs the MIO OUTSEL CSRs to to ensure that in deep sleep it randomly picks
+            between tie-0, tie-1 or hi-Z for all muxed outputs coming from non-AON IPs. If an AON
+            peripheral output is muxed, then that peripheral's output is selected to ensure in deep
+            sleep the peripheral can continue its signaling even in deep sleep. The testbench
+            verifies the correctness of the reflected values once the chip goes into deep sleep.
+            This is replicated for DIO pins as well.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sleep_pin_wake
+      desc: '''Verify pin wake up from deep sleep state.
+
+            Verify one of the 8 possible MIO or DIO pad inputs (randomly configured) can cause the
+            chip to wake up from sleep state. Verifying wake on posedge is sufficient for the chip
+            level integration testing. Upon wake up, SW reads the wake cause CSR to verify
+            correctness.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // PADCTRL tests:
+    {
+      name: chip_padctrl_attributes
+      desc: '''Verify pad attribute settings for all MIO and DIO pads.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // RV_DM (JTAG) tests:
```

Comment:
one thing we should add (it's sort one level above this one, and technically more Nuvoton's responsbility), is a test to select between all the TAP points.  We currently plan to have 3.